### ISSUE TITLE
TraitUseSpacing false-positives/requests

### ISSUE THREAD

**Issue by Levivb** [slevomat/coding-standard#402]
I encountered a kinda false positive in the default Laravel (5.2) installation for a required trailing newline after a `use Trait;` statement.

See: https://github.com/laravel/laravel/blob/5.2/app/Jobs/Job.php

```php
abstract class Job
{
    /*
    |--------------------------------------------------------------------------
    | Queueable Jobs
    |--------------------------------------------------------------------------
    |
    | This job base class provides a central location to place any logic that
    | is shared across all of your jobs. The trait included with the class
    | provides access to the "onQueue" and "delay" queue helper methods.
    |
    */
    use Queueable;
}
```

Entering a newline conflicts with `PSR2.Classes.ClassDeclaration.CloseBraceAfterBody`

Would it be possible to create a newline exception for a trailing closing body brace?

**Comment by kukulich** [comment]
I've just added new option: https://github.com/slevomat/coding-standard/commit/90511d98c49bfe0e50832452443d22ff2d4656e4

**Comment by Levivb** [comment]
Cool! That speed though!
Love the amount of Sniffs your putting together here!
